components/Cards/SchoolCard/FrontSchoolCard.js

```diff
@@ -0,0 +1,140 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import Button from 'components/_common_/Button/Button';
+import OutboundLink from 'components/_common_/OutboundLink/OutboundLink';
+import DesktopIcon from 'static/images/icons/FontAwesome/desktop-solid.svg';
+import DevicesIcon from 'static/images/icons/FontAwesome/devices-alt-solid.svg';
+import HomeIcon from 'static/images/icons/FontAwesome/home-solid.svg';
+import PeopleIcon from 'static/images/icons/FontAwesome/users-solid.svg';
+import GIBillApprovedIcon from 'static/images/icons/gi-bill-approved.svg';
+import GIBillUnavailableIcon from 'static/images/icons/gi-bill-unavailable.svg';
+import Badge from 'components/Badge/Badge';
+import styles from './FrontSchoolCard.css';
+
+export default class FrontSchoolCard extends Component {
+  static propTypes = {
+    cardFlipCallback: PropTypes.func.isRequired,
+    hasHardwareIncluded: PropTypes.bool.isRequired,
+    hasHousing: PropTypes.bool.isRequired,
+    hasOnline: PropTypes.bool.isRequired,
+    hasOnlyOnline: PropTypes.bool.isRequired,
+    isFullTime: PropTypes.bool.isRequired,
+    locations: PropTypes.arrayOf(
+      PropTypes.shape({
+        city: PropTypes.string,
+        va_accepted: PropTypes.bool.isRequired,
+        state: PropTypes.string,
+      }),
+    ).isRequired,
+    logoSource: PropTypes.string.isRequired,
+    name: PropTypes.string.isRequired,
+    website: PropTypes.string.isRequired,
+  };
+
+  render() {
+    const { props } = this;
+
+    return (
+      <>
+        <h5>{props.name}</h5>
+        <img src={props.logoSource} alt="" />
+
+        <div className={styles.interactionsContainer}>
+          <Button>
```

Comment:
Let's drop `<Button><OutboundLink>...` here and use the new `<LinkButton>` component.